src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -3,7 +3,52 @@
 <head>
     <meta charset="utf-8">
     <title>{0}</title>
+    <script>  
+      (function() {{
+          const logPrefix = '[Starcounter App Shell]';
+          function register() {{
+          if ('serviceWorker' in navigator) {{
+            window.addEventListener('load', () => {{
+              const swUrl = `/service-worker.js`;
+              registerSW(swUrl);
```

Comment:
This script contains two functions with a very similar name: `register` and `registerSW`. Can you think of better names please?

Also, the indentation is broken in this script.